openslides_backend/action/action.py

```diff
@@ -393,7 +392,12 @@ def validate_relation_fields(self, instance: Dict[str, Any]) -> None:
             for equal_field in field.equal_fields:
                 if not (own_equal_field_value := instance.get(equal_field)):
                     fqid = FullQualifiedId(self.model.collection, instance["id"])
-                    db_instance = self.fetch_model(fqid, [equal_field])
+                    db_instance = self.datastore.fetch_model(
+                        fqid,
+                        [equal_field],
+                        db_additional_relevance=InstanceAdditionalBehaviour.ADDITIONAL_BEFORE_DBINST,
```

Comment:
since `ADDITIONAL_BEFORE_DBINST` is the default, I would suggest to remove it here and everywhere else where it isn't needed. Makes the code more readable and less confusing since you don't have to bother with any db-instance-handling if it's the default case.

Reply:
There would be more defaults to change this way. But if we want to replace the get-method by the fetch.model-method we should use here the defaults from get.